python/pyspark/errors/__init__.py

```diff
@@ -18,9 +18,29 @@
 """
 PySpark exceptions.
 """
-from pyspark.errors.exceptions import PySparkException
```

Comment:
FYI: The changes here and `python/pyspark/errors/exceptions.py` will come from https://github.com/apache/spark/pull/39591 as well.

I manually update these file in advance to manage the test cases.